test/seed/852-rule.rule.merge.yaml

```diff
@@ -131,10 +144,24 @@ splitName: # defaults name: fee, tag: |fee|acquirer|
   - {conditionName: Test Split Range, name: issuer, tag: "|issuer|"}
   - {conditionName: Test Split Analytic}
   - {conditionName: Test Split Analytic, name: pos, tag: "|pos|"}
+  - {conditionName: Rule Test Limits}
+  - {conditionName: Rule Test More Assignments, name: testFee}
+  - {conditionName: Rule Test More Assignments, name: base}
```

Comment:
The split named `base` usually is not tagged with `|fee|`, because it uses percent=100 and so it means 100% fee, which can be observed as weird `acquirerFee` amounts in the snapshots (i.e. amounts bigger than the transfer amount). The `|fee|` tag is set by default, when the tag is not provided. I suggest all splits named `base` to have tag `|issuer|`.

Reply:
I added |issuer| tag for the splits with name "base". The acquirerFee amount is now the same as the fee amount.